src/tools/clippy/tests/ui/option_if_let_else.fixed

```diff
@@ -22,22 +22,14 @@ fn else_if_option(string: Option<&str>) -> Option<(bool, &str)> {
     }
 }
 
-fn unop_bad(string: &Option<&str>, mut num: Option<i32>) {
+fn unop_bad(string: &Option<&str>, num: Option<i32>) {
     let _ = string.map_or(0, |s| s.len());
     let _ = num.as_ref().map_or(&0, |s| s);
-    let _ = num.as_mut().map_or(&mut 0, |s| {
```

Comment:
This PR removes tests from Clippy. Why is that?

Reply:
This is a remnant of a previous version of https://github.com/rust-lang/rust/pull/102256. Fixed.